README.md

```diff
@@ -21,14 +21,19 @@ The site will be available at [http://localhost:4000](http://localhost:4000).
 ## Team Leads: Add or Update your project in the Yearbook
 
 **Template for Project**
-  ## Title of the Project
+
+  Title of the Project
+
   Description of the project and the organization the project was built for. If possible, mention the impact the project has had for the original organization or others who have begun using it.
+  
   * List of team members on the project team
   * Member 1
   * Member 2
   * Member 3
   * etc.
+
```

Comment:
Do we want to describe what this stuff that's currently in the yearbook page is doing? <img width="717" alt="screen shot 2016-06-18 at 5 33 50 pm" src="https://cloud.githubusercontent.com/assets/7772827/16173788/e7746f74-357a-11e6-8b88-1a5eff711e3a.png">